components/support/SupportLayerItemFire.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+    <l-lwms-tile-layer
+        v-if="layer.layer_type == 'wms'"
+        ref="wmsLayer"
+        format="image/png"
+        :base-url="wmsBaseUrl"
+        :layers="geoserverLayer"
+        :transparent="true"
+        :z-index="3"
+        :visible="layer.visible"
+        :options="wmsOptions"
+    />
+
+    <l-tile-layer
+        v-else-if="layer.layer_type == 'tms'"
+        :url="layer.tms.url_tms"
+        :visible="layer.visible"
+        :z-index="3"
+        :options="{
+            maxNativeZoom: layer.tms.max_native_zoom
+                ? layer.tms.max_native_zoom
+                : 15,
+        }"
+    />
+
+    <l-layer-group
+        v-else-if="layer.layer_type == 'heatmap'"
+        ref="heatmap"
+        :visible="layer.visible"
+    />
+</template>
+
+<script>
+import { mapMutations, mapActions } from 'vuex'
+
+if (typeof window !== 'undefined') {
+    require('leaflet.heat')
+}
+
+export default {
+    name: 'SupportLayerItemFire',
+
+    props: {
+        layer: {
+            type: Object,
+            required: true,
+        },
+    },
+
+    computed: {
+        defaultFilters() {
+            const filters = {}
+
+            this.layer.layer_filters.forEach((layerFilter) => {
+                const label = layerFilter.filter_type
+                const value = layerFilter.default
+                filters[label] = value
+            })
+
+            return filters
+        },
+
+        geoserverLayer() {
+            if (
+                this.layer.wms &&
+                this.layer.wms.geoserver_layer_namespace &&
+                this.layer.wms.geoserver_layer_name
+            ) {
+                return (
+                    this.layer.wms.geoserver_layer_namespace +
+                    ':' +
+                    this.layer.wms.geoserver_layer_name
+                )
+            } else return ''
+        },
+
+        wmsOptions() {
+            let options = {}
+            if (this.layer.layer_type === 'wms') {
+                options = {
+                    name: this.layer.name,
+                    maxZoom: 21,
+                    maxNativeZoom: 19,
+                    queryable: this.layer.wms.queryable,
+                }
+
+                if (this.layer.wms && this.layer.wms.geoserver_layer_options) {
+                    options = {
+                        ...options,
+                        ...this.layer.wms.geoserver_layer_options,
+                    }
+                }
+            }
+
+            return options
+        },
+
+        wmsBaseUrl() {
```

Comment:
this method have to be on SupportLayerItem too, it has no diffs with another layers.

Reply:
Resolved